modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java

```diff
@@ -362,6 +378,18 @@ public Server getServer(ObjectNode obj, String location, ParseResult result) {
 
         String value = getString("url", obj, true, location, result);
         if(StringUtils.isNotBlank(value)) {
+        	File file = new File(value);
+			if(!isValidURL(value) && !file.isAbsolute() && path != null){
```

Comment:
The check `&& !file.isAbsolute()` is not necessary if you consider a server definition like this:

```json
      {
        "description": "Server relative to root path",
        "url": "/api/v2"
      }
```

If you remove the check, then you also can remove `File file = new File(value);`